tests/functional/aws-node-sdk/test/multipleBackend/put.js

```diff
@@ -0,0 +1,181 @@
+import assert from 'assert';
+import withV4 from '../support/withV4';
+import BucketUtility from '../../lib/utility/bucket-util';
+import config from '../../../../../lib/Config';
+
+const bucket = 'buckettestmultiplebackendput';
+const key = 'somekey';
+const body = 'somestring';
+let bucketUtil;
+let s3;
+
+
+describe('MultipleBackend put object', () => {
+    withV4(sigCfg => {
+        beforeEach(() => {
+            bucketUtil = new BucketUtility('default', sigCfg);
+            s3 = bucketUtil.s3;
+            process.stdout.write('Creating bucket');
+            return s3.createBucketAsync({ Bucket: bucket })
+            .catch(err => {
+                process.stdout.write(`Error creating bucket: ${err}\n`);
+                throw err;
+            });
+        });
+
+        afterEach(() => {
+            process.stdout.write('Emptying bucket\n');
+            return bucketUtil.empty(bucket)
+            .then(() => {
+                process.stdout.write('Deleting bucket\n');
+                return bucketUtil.deleteOne(bucket);
+            })
+            .catch(err => {
+                process.stdout.write(`Error in afterEach: ${err}\n`);
+                throw err;
+            });
+        });
+
+        it('should return an error to put request without a valid bucket name',
+            done => {
+                s3.putObject({ Bucket: '', Key: key }, err => {
+                    assert.notEqual(err, null,
+                        'Expected failure but got success');
+                    assert.strictEqual(err.code, 'MethodNotAllowed');
+                    done();
+                });
+            });
+
+        it('should return an error to put request without a valid key name',
```

Comment:
I'd remove this test.  It's really just making a put bucket call since there is no key.  You would get a bucketAlreadyOwnedByYou error if not in us-east-1 and the bucket is already owned by you and otherwise the bucket would actually be put.